c10/cuda/driver_api.cpp

```diff
@@ -8,19 +8,30 @@ namespace cuda {
 
 namespace {
 DriverAPI create_driver_api() {
-#define OPEN_LIBRARIES(name, n)               \
-  void* handle_##n = dlopen(name, RTLD_LAZY); \
-  TORCH_INTERNAL_ASSERT(handle_##n);
+  void* handle_0 = dlopen("libcuda.so", RTLD_LAZY | RTLD_NOLOAD);
+  TORCH_INTERNAL_ASSERT(handle_0);
+  void* handle_1 = dlopen("libnvidia-ml.so.1", RTLD_LAZY);
+  try {
+    TORCH_INTERNAL_ASSERT(handle_1);
+  } catch (...) {
```

Comment:
Is it necessary to have a catch (all) here?

Reply:
Followed your offline suggestion to change this to a simple if check, thanks.